ArcadiaTeamsBot.ServiceDesk.Requests/RequestTypeForUIField.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ArcadiaTeamsBot.ServiceDesk.Requests
+{
+    public class RequestTypeForUIField
+    {
+        public RequestTypeForUIField(string fieldName, RequestTypeForUIFieldType fieldType)
+        {
+            this.FieldName = fieldName;
+            this.FieldType = fieldType;
+        }
+        public string FieldName { get; set; }
```

Comment:
...

Reply:
Sorry, forgot to autoformat.
Already fixed.